spec/models/post_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(author_id: 1, title: 'My first post', body: 'This is my first post') }
+
+  before { subject.save }
+
+  it 'Validates the presence of Title' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'Validates comment counter to not be a string' do
+    subject.comment_count = 'great'
+    expect(subject).to_not be_valid
+  end
+
+  it 'Validates the length of Title to be less than or equal to 250' do
+    subject.title = 'a' * 251
+    expect(subject).to_not be_valid
+  end
+
+  it 'Validates increament of post counter after creation of post' do
+    user = User.create(name: 'Philip', photo: 'https://...', bio: 'physician from Britain')
+    expect(user.post_count).to eq(1)
+  end
+end
```

Comment:
- [ ] One of the major project requirements is to write unit tests for all corresponding custom methods in each model. Kindly write a corresponding unit test for the `recent_comments` method in your `Post` model. 👍